### PR TITLE
アイテム追加後にショッピングカート一覧がみれない現象の改善

### PR DESCRIPTION
**Pull request by tkeshun**

### CODE REVIEW

File: src/main/java/com/example/controller/ShoppingCartController.java [hunk 1]
```diff
@@ -40,6 +40,11 @@ public String showCart(Model model, @AuthenticationPrincipal LoginUser loginUser
 		User user = null;
 		if (loginUser != null) {
 			user = loginUser.getUser();
+			if (user == null) {
```
Comment (tkeshun) [inline src/main/java/com/example/controller/ShoppingCartController.java:43]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (tkeshun) [inline src/main/java/com/example/controller/ShoppingCartController.java:43]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'if' has incorrect indentation level 24, expected level should be 6.

File: src/main/java/com/example/controller/ShoppingCartController.java [hunk 2]
```diff
@@ -40,6 +40,11 @@ public String showCart(Model model, @AuthenticationPrincipal LoginUser loginUser
...
+				user = (User) httpSession.getAttribute("currentUser");
```
Comment (tkeshun) [inline src/main/java/com/example/controller/ShoppingCartController.java:44]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (tkeshun) [inline src/main/java/com/example/controller/ShoppingCartController.java:44]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'if' child has incorrect indentation level 32, expected level should be 8.

File: src/main/java/com/example/controller/ShoppingCartController.java [hunk 3]
```diff
@@ -40,6 +40,11 @@ public String showCart(Model model, @AuthenticationPrincipal LoginUser loginUser
...
+			} else {
```
Comment (tkeshun) [inline src/main/java/com/example/controller/ShoppingCartController.java:45]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (tkeshun) [inline src/main/java/com/example/controller/ShoppingCartController.java:45]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'if rcurly' has incorrect indentation level 24, expected level should be 6.

File: src/main/java/com/example/controller/ShoppingCartController.java [hunk 4]
```diff
@@ -40,6 +40,11 @@ public String showCart(Model model, @AuthenticationPrincipal LoginUser loginUser
...
+				httpSession.setAttribute("currentUser", user);
```
Comment (tkeshun) [inline src/main/java/com/example/controller/ShoppingCartController.java:46]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (tkeshun) [inline src/main/java/com/example/controller/ShoppingCartController.java:46]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'else' child has incorrect indentation level 32, expected level should be 8.

File: src/main/java/com/example/controller/ShoppingCartController.java [hunk 5]
```diff
@@ -40,6 +40,11 @@ public String showCart(Model model, @AuthenticationPrincipal LoginUser loginUser
...
+			}
```
Comment (tkeshun) [inline src/main/java/com/example/controller/ShoppingCartController.java:47]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (tkeshun) [inline src/main/java/com/example/controller/ShoppingCartController.java:47]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'else rcurly' has incorrect indentation level 24, expected level should be 6.

File: src/main/java/com/example/controller/ShoppingCartController.java [hunk 6]
```diff
@@ -79,19 +85,25 @@ public String showCart(Model model, @AuthenticationPrincipal LoginUser loginUser
 		model.addAttribute("cartItem", showShoppingCartContents);
 		model.addAttribute("totalTax", order.getTax());
 		model.addAttribute("totalPrice", order.getCalcPrice());
-		//追加
+		// 追加
```
Comment (tkeshun) [inline src/main/java/com/example/controller/ShoppingCartController.java:88]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

File: src/main/java/com/example/controller/ShoppingCartController.java [hunk 7]
```diff
@@ -79,19 +85,25 @@ public String showCart(Model model, @AuthenticationPrincipal LoginUser loginUser
...
 		model.addAttribute("order", order);
 		return "ec/cart_list";
 	}
 
 	@PostMapping("/add-item")
-	public String addItem(AddItemToShoppingCartForm form, RedirectAttributes redirectAttributes) {
-		System.out.println(form);
+	public String addItem(AddItemToShoppingCartForm form, RedirectAttributes redirectAttributes,
```
Comment (tkeshun) [inline src/main/java/com/example/controller/ShoppingCartController.java:94]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

File: src/main/java/com/example/controller/ShoppingCartController.java [hunk 8]
```diff
@@ -79,19 +85,25 @@ public String showCart(Model model, @AuthenticationPrincipal LoginUser loginUser
...
+			@AuthenticationPrincipal LoginUser loginUser) {
```
Comment (tkeshun) [inline src/main/java/com/example/controller/ShoppingCartController.java:95]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

File: src/main/java/com/example/controller/ShoppingCartController.java [hunk 9]
```diff
@@ -79,19 +85,25 @@ public String showCart(Model model, @AuthenticationPrincipal LoginUser loginUser
...
 		if (form.getQuantity() == null) {
 			redirectAttributes.addFlashAttribute("qerr", "数量を選択してください");
 			return "redirect:/show-detail/show-item-detail?itemId=" + form.getItemId();
 		}
-		User user = (User) httpSession.getAttribute("currentUser");
+		User user = null;
```
Comment (tkeshun) [inline src/main/java/com/example/controller/ShoppingCartController.java:100]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (tkeshun) [inline src/main/java/com/example/controller/ShoppingCartController.java:100]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def' child has incorrect indentation level 16, expected level should be 4.

File: src/main/java/com/example/controller/ShoppingCartController.java [hunk 10]
```diff
@@ -79,19 +85,25 @@ public String showCart(Model model, @AuthenticationPrincipal LoginUser loginUser
...
+		if (loginUser != null) {
```
Comment (tkeshun) [inline src/main/java/com/example/controller/ShoppingCartController.java:101]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (tkeshun) [inline src/main/java/com/example/controller/ShoppingCartController.java:101]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'if' has incorrect indentation level 16, expected level should be 4.

File: src/main/java/com/example/controller/ShoppingCartController.java [hunk 11]
```diff
@@ -79,19 +85,25 @@ public String showCart(Model model, @AuthenticationPrincipal LoginUser loginUser
...
+			user = loginUser.getUser();
```
Comment (tkeshun) [inline src/main/java/com/example/controller/ShoppingCartController.java:102]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (tkeshun) [inline src/main/java/com/example/controller/ShoppingCartController.java:102]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'if' child has incorrect indentation level 24, expected level should be 6.

File: src/main/java/com/example/controller/ShoppingCartController.java [hunk 12]
```diff
@@ -79,19 +85,25 @@ public String showCart(Model model, @AuthenticationPrincipal LoginUser loginUser
...
+
+		} else {
```
Comment (tkeshun) [inline src/main/java/com/example/controller/ShoppingCartController.java:104]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (tkeshun) [inline src/main/java/com/example/controller/ShoppingCartController.java:104]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'if rcurly' has incorrect indentation level 16, expected level should be 4.

Comment (tkeshun) [inline src/main/java/com/example/controller/ShoppingCartController.java:104]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.blocks.EmptyBlockCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Empty else block.

File: src/main/java/com/example/controller/ShoppingCartController.java [hunk 13]
```diff
@@ -79,19 +85,25 @@ public String showCart(Model model, @AuthenticationPrincipal LoginUser loginUser
...
+
+		}
```
Comment (tkeshun) [inline src/main/java/com/example/controller/ShoppingCartController.java:106]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (tkeshun) [inline src/main/java/com/example/controller/ShoppingCartController.java:106]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'else rcurly' has incorrect indentation level 16, expected level should be 4.

File: src/main/java/com/example/repository/OrderRepository.java [hunk 14]
```diff
@@ -60,7 +60,14 @@ public class OrderRepository {
 				orderItem.setItemId(rs.getInt("oItem_item_id"));
 				orderItem.setOrderId(rs.getInt("oItem_order_id"));
 				orderItem.setQuantity(rs.getInt("oItem_quantity"));
-				orderItem.setSize(rs.getString("oItem_size").charAt(0));
+				String string = rs.getString("oItem_size");
```
Comment (tkeshun) [inline src/main/java/com/example/repository/OrderRepository.java:63]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (tkeshun) [inline src/main/java/com/example/repository/OrderRepository.java:63]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'if' child has incorrect indentation level 32, expected level should be 14.

File: src/main/java/com/example/repository/OrderRepository.java [hunk 15]
```diff
@@ -60,7 +60,14 @@ public class OrderRepository {
...
+
+				if (string != null) {
```
Comment (tkeshun) [inline src/main/java/com/example/repository/OrderRepository.java:65]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (tkeshun) [inline src/main/java/com/example/repository/OrderRepository.java:65]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'if' has incorrect indentation level 32, expected level should be 14.

File: src/main/java/com/example/repository/OrderRepository.java [hunk 16]
```diff
@@ -60,7 +60,14 @@ public class OrderRepository {
...
+					char[] c = string.toCharArray();
```
Comment (tkeshun) [inline src/main/java/com/example/repository/OrderRepository.java:66]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (tkeshun) [inline src/main/java/com/example/repository/OrderRepository.java:66]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'if' child has incorrect indentation level 40, expected level should be 16.

**Review by tkeshun — COMMENTED** [review]
Remaining comments which cannot be posted as a review comment to avoid GitHub Rate Limit

<details>
<summary>checkstyle</summary>

[src/main/java/com/example/repository/OrderRepository.java|67 col 1|](http://github.com/ecsite-project/ec-202304c/blob/cfd0d7a2a7fae7b09da8184e47fa71813a488a10/src/main/java/com/example/repository/OrderRepository.java#L67) Line contains a tab character.
[src/main/java/com/example/repository/OrderRepository.java|67 col 41|](http://github.com/ecsite-project/ec-202304c/blob/cfd0d7a2a7fae7b09da8184e47fa71813a488a10/src/main/java/com/example/repository/OrderRepository.java#L67) 'if' child has incorrect indentation level 40, expected level should be 16.
[src/main/java/com/example/repository/OrderRepository.java|68 col 1|](http://github.com/ecsite-project/ec-202304c/blob/cfd0d7a2a7fae7b09da8184e47fa71813a488a10/src/main/java/com/example/repository/OrderRepository.java#L68) Line contains a tab character.
[src/main/java/com/example/repository/OrderRepository.java|68 col 33|](http://github.com/ecsite-project/ec-202304c/blob/cfd0d7a2a7fae7b09da8184e47fa71813a488a10/src/main/java/com/example/repository/OrderRepository.java#L68) 'if rcurly' has incorrect indentation level 32, expected level should be 14.
[src/main/java/com/example/repository/OrderRepository.java|69 col 1|](http://github.com/ecsite-project/ec-202304c/blob/cfd0d7a2a7fae7b09da8184e47fa71813a488a10/src/main/java/com/example/repository/OrderRepository.java#L69) Line contains a tab character.
[src/main/java/com/example/repository/OrderRepository.java|69 col 41|](http://github.com/ecsite-project/ec-202304c/blob/cfd0d7a2a7fae7b09da8184e47fa71813a488a10/src/main/java/com/example/repository/OrderRepository.java#L69) 'else' child has incorrect indentation level 40, expected level should be 16.
[src/main/java/com/example/repository/OrderRepository.java|70 col 1|](http://github.com/ecsite-project/ec-202304c/blob/cfd0d7a2a7fae7b09da8184e47fa71813a488a10/src/main/java/com/example/repository/OrderRepository.java#L70) Line contains a tab character.
[src/main/java/com/example/repository/OrderRepository.java|70 col 33|](http://github.com/ecsite-project/ec-202304c/blob/cfd0d7a2a7fae7b09da8184e47fa71813a488a10/src/main/java/com/example/repository/OrderRepository.java#L70) 'else rcurly' has incorrect indentation level 32, expected level should be 14.
</details>

**Review by yousuke-murayama — APPROVED** [review]
問題ないと思います。

**Pull request merged**